Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 [dependencies]
 anyhow = "1"
 clap = { version = "2.33", default-features = false }
-rust-ini = "0.13"
+rust-ini = "0.15"
```

Comment:
ehh... this will bring quite a lot of dependencies

Reply:
Hmm, my thinking was that this will need to be done at some point, and it's better to update before adding new code that touches the same paths. What would the plan be? To always keep on the old version?

Reply:
I wanted to bring upstream attention to this problem, but did not have time yet.

Reply:
given that it is essentially drop-in replacement with your PR (I mean changing to 0.13 still compiles and tests pass), I think it is fine to keep whatever you like. In Fedora we can just revert this version bump.

Reply:
```
└── rust-ini v0.15.3
    ├── cfg-if v0.1.10
    └── ordered-multimap v0.2.4
        ├── dlv-list v0.2.2
        │   └── rand v0.7.3
        │       ├── getrandom v0.1.14
        │       │   ├── cfg-if v0.1.10
        │       │   └── libc v0.2.71
        │       ├── libc v0.2.71
        │       ├── rand_chacha v0.2.2
        │       │   ├── ppv-lite86 v0.2.8
        │       │   └── rand_core v0.5.1
        │       │       └── getrandom v0.1.14 (*)
        │       └── rand_core v0.5.1 (*)
        └── hashbrown v0.7.2
            └── ahash v0.3.8
            [build-dependencies]
            └── autocfg v1.0.0
```

It is basically this big list of dependencies. That is twice bigger than all other dependencies and their recursive dependencies too.